prework.md

```diff
@@ -0,0 +1,17 @@
+## My questions from the reading
+### Heap
+* "In order to maintain the heap order property, all we need to do is swap the root with its smallest child less than the root." why the smallest child less than the root? what happens if we swap it with the larger child?
+* Given the same list of numbers - will the binary heap list be exactly the same regardless of the order in which the numbers are inserted into the heap?
+* "Although we start out in the middle of the tree and work our way back toward the root, the percolate_down method ensures that the largest child is always moved down the tree." I'm not sure what they mean by largest child. largest child of...?
```

Comment:
I think the easiest way to clear this one up is to do an example (either during or after class).